### DISCUSSION TITLE
Bagaimana Cara Knit.Html

### DISCUSSION THREAD

**Discussion by lonexindo** [teamalgoritma/community#667] [Q&A] [answered]
Saya lagi berlatih untuk membuat contingency table, tapi waktu mau kumpulin tugas yang mengharuskan saya meng-klik Knit.Html tapi ga bisa dengan tampilan seperti ini:

<img width="960" alt="image" src="https://github.com/teamalgoritma/community/assets/133940981/ebc749f1-15ef-4afb-be71-cba4cdb831d8">

**Comment by nadyadtm** [comment]
Dear @lonexindo 

Apakah untuk kasus ini sudah terselesaikan?

**Comment by lonexindo** [comment]
Selamat siang mba Nadya,

Untuk kasus ini saya sudah coba selesaikan. Saya sudah informasi pak Irfan
untuk Link-nya.

https://rpubs.com/lonexindo/1043096

Berikut adalah link-nya. Saya sudah coba buka dan berhasil. Bisa tolong di
cross check apakah sudah sesuai?

terima kasih,
Hari

**Comment by lonexindo** [comment]
Selamat siang mba Nadya

Apakah sudah ada feedback terkait hasil LBB workshop kemaren?

Untuk link-nya apakah sudah berhasil dibuka?

Sebab saya belum mendapat feedback dari pak Irfan juga.

Terima kasih,
Hari

**Comment by nadyadtm** [comment, marked as answer]
Dear Pak @lonexindo 
Mohon maaf sebelumnya pak, Untuk hasil LBBnya sedang diperiksa terlebih dahulu. Akan kami kabarkan bila sudah selesai. Namun untuk link Rpubsnya seharusnya sudah aman.

**Comment by lonexindo** [comment]
Dear Lonex Indonesia,    Mohon maaf sebelumnya pak, Untuk hasil LBBnya sedang diperiksa terlebih dahulu. Akan kami kabarkan bila sudah selesai. Namun untuk link Rpubsnya seharusnya sudah aman.      Best Regards,  Team Algoritma        
                           
                            On
                            Tue, 23 May at  2:54 PM
                            ,  Lonex Indonesia ***@***.***>  wrote:
                             
 Selamat siang mba Nadya 
   
 Apakah sudah ada feedback terkait hasil LBB workshop kemaren? 
   
 Untuk link-nya apakah sudah berhasil dibuka? 
   
 Sebab saya belum mendapat feedback dari pak Irfan juga. 
   
 Terima kasih, 
 Hari 
 
 
  
 On Sun, May 21, 2023 at 3:32 PM Lonex Indonesia ***@***.***> wrote: 
 
 
 Selamat siang mba Nadya,   
 Untuk kasus ini saya sudah coba selesaikan. Saya sudah informasi pak Irfan untuk Link-nya. 
   
 
https://rpubs.com/lonexindo/1043096 
 
   
 Berikut adalah link-nya. Saya sudah coba buka dan berhasil. Bisa tolong di cross check apakah sudah sesuai? 
   
 terima kasih, 
 Hari 
 
  
 On Fri, May 19, 2023 at 5:05 PM Nadya Aditama ***@***.***> wrote: 
 
 
 
 Dear @lonexindo
 
 Apakah untuk kasus ini sudah terselesaikan? 

 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>
 
 
 
 
 

 

--  
To unsubscribe from this group and stop receiving emails from it, send an email to ***@***.***
